presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -352,6 +352,72 @@ as a part of a SQL query by appending name to the table.
     ---------+------------------------------------------------------------------------------+-------------+--------------+--------------------+-----------------------------+--------------------------+----------------------+------------------+-------------------------------------------+--------------------------------------------+--------------+---------------+-------------
        0     | s3://my-bucket/ctas_nation/data/9f889274-6f74-4d28-8164-275eef99f660.parquet | PARQUET     |           25 |               1648 | {1=52, 2=222, 3=105, 4=757} | {1=25, 2=25, 3=25, 4=25} | {1=0, 2=0, 3=0, 4=0} |  NULL            | {1=0, 2=ALGERIA, 3=0, 4= haggle. careful} | {1=24, 2=VIETNAM, 3=4, 4=y final packaget} | NULL         | NULL          | NULL
 
+``$changelog`` Table
+^^^^^^^^^^^^^^^^^^^^
+
+This table lets you view which row-level changes which have occurred to the
+table in a particular order over time. Represents the history of changes to the
+table, while also making the raw data available to query.
+
+The result of a changelog query always returns a static schema with 4 columns:
+
+1. ``operation``: (``VARCHAR``) indicating whether the row was inserted,
+   updated, or deleted.
+2. ``ordinal``: (``int``) A number indicating a relative order that a particular
+   change needs to be applied to the table relative to all other changes.
+3. ``snapshotid``: (``bigint``) Represents the snapshot a row-level
+   change was made in.
+4. ``rowdata``: (``row(T)``) which includes the data for the particular row. The
+   inner values of this type match the schema of the parent table.
+
+The changelog table can be queried with the following name format:
+
+.. code-block:: sql
+
+    ... FROM "<table>@<begin snapshot ID>$changelog@[<end snapshot ID>]"
+
+- ``<table>`` is the name of the table.
+- ``<begin snapshot ID>`` is the snapshot of the table you want to begin viewing
+  changes from.
+- ``<end snapshot ID>`` is the last snapshot for which you want to view changes.
+  This parameter is optional. If left out, the most current snapshot of the
+  table is used.
+
```

Comment:
I was a little confused here, let me explain why. 
You wrote "The result of a changelog query always returns a static schema with 4 columns:"
and this first example I find six columns, not all of which match the four. 

Rereading I realize that this six-column example is about finding the start and optional ending snapshot ID that are then used in the actual changelog query. Do I understand this correctly?

Maybe, after the sentence "If left out, the most current snapshot of the table is used." you could add an explanatory introduction here, something like

"To find the snapshot_id value or values, run a command similar to the following:

SELECT * FROM "<table>$snapshots";

For example, to query for the snapshots of the orders table, run the following command: "

My example isn't formatted fully, so please use or adapt it and format such text as "snapshot_id" and "orders" as inline code.

Reply:
I agree with your analysis the text didn't describe why we queried the `$snapshots` table first. I added a few sentences beforehand to describe the example first. Please let me know if this helps clarify!

Reply:
Looks great, thanks!

Reply:
This definitely clarifies the description. Nice work!